tests/network-setups/indy/docker/Makefile

```diff
@@ -0,0 +1,11 @@
+image:
+	docker build -t testiin .
+
+start:
+	./start_testiin.sh
+
+stop:
+	./stop_testiin.sh
+
+clean:
```

Comment:
Add `stop` to this as a target to complete before cleanup (i.e., after the colon). We shouldn't try to cleanup without stopping the network first.

Reply:
I'll add this in my next PR.